homeassistant/config.py

```diff
@@ -1015,21 +1029,115 @@ async def merge_packages_config(
     return config
 
 
+@callback
+def async_process_component_config_errors(
+    hass: HomeAssistant,
+    integration: Integration,
+    config_exception_info: list[ConfigExceptionInfo],
```

Comment:
This is a list, plural is clearer imo
```suggestion
    config_exception_infos: list[ConfigExceptionInfo],
```